core/server/index.js

```diff
@@ -179,7 +179,9 @@ const isDatabaseInitialisationRequired = () => {
                     });
             }
         })
-        .then(() => {
+        .then(async () => {
+            await db.destroy();
+            await db.initialize();
```

Comment:
I don't know exactly why this is necessary but without it [the columnInfo query](https://github.com/TryGhost/Ghost/pull/11962/files#diff-c92d0218d487c1b8b8fe495c4c25afffR232) returns an empty result set.

Reply:
Had to be moved to `populateDefaults()` for tests with teardown/setup to work correctly